env_cleaner.py

```diff
@@ -33,6 +34,35 @@
 SANDBOX_MATCHER = re.compile(".+-sandbox$")
 SANDBOX_ANTIMATCHER = re.compile(".+-internal-qa-sandbox")
 
+# Map of repository names to service names
+REPO_NAMES = {
+    "personal-demographics-service-api": "personal-demographics",
```

Comment:
Presumably we would need to add the EPS repo and base path into here?

Reply:
Correct